bootstrap

```diff
@@ -63,4 +63,12 @@ echo "Generating file lists: ${VARS_FILE}"
   printf "TSS2_MU_SRC = \$(TSS2_MU_C) \$(TSS2_MU_H)"
 ) > ${VARS_FILE}
 
+# Generate fuzz tests unless environment variable NO_GEN_FUZZ is set to 1
+if test "${NO_GEN_FUZZ}0" -ne 10; then
```

Comment:
"x${ABC}" -ne "x1" is way more common than appending a 0...

Reply:
That doesn't seem to work, I think because we are in sh instead of bash. Okay if i leave this way?